### PR TITLE
Enable dropdown on hover

### PR DESCRIPTION
**Pull request by RAJAGOPALAN-GANGADHARAN**
Fixed enable auto hover
The other previous commits ignore pls.Will work on a seperate branch from next time

### CODE REVIEW

File: interface/forms/LBF/view.php [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <?php
 require("new.php");
-?>
\ No newline at end of file
+?>
```
Comment (aethelwulffe) [inline interface/forms/LBF/view.php]: Nice!

File: interface/new/new.php [hunk 2]
```diff
@@ -1,242 +1,3 @@
...
-            timepicker: false,
-            format: "<?= $DateFormat; ?>"
-        });
-        $.datetimepicker.setLocale('<?= $DateLocale;?>');
-    });
-</script>
-
-</body>
-</html>
+//Useless
```
Comment (aethelwulffe) [inline interface/new/new.php]: You preserved this file by reducing it to two lines?  Is this related to this PR at all?
-I will go back and read your description.  It looks like you may need to rebase your fork, bringing it up to sync with the head, then create a new branch off of your nice, clean, rebased master for this.  I believe it has changes from your previous commits.
  I will test the changes to the menu hover.

**Comment by aethelwulffe** [comment]
Re-read your post.  Now I understand.  Gotcha.

You need to rebase your master remote fork, then sync your local, recreate your branch, then never ever ever push a commit to your master branch again.  :)

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
@aethelwulffe hmm i understand
will fix it within some time.Thank you for the support

File: interface/patient_file/encounter/new_form.php [hunk 3]
```diff
@@ -189,7 +183,7 @@ function myGetRegistered($state="1", $limit="unlimited", $offset="0") {
           $new_category_ = $new_category;
           $new_category_ = str_replace(' ','_',$new_category_);
           if ($old_category != '') {$StringEcho.= "</table></div></li>";}
-          $StringEcho.= "<li class=\"encounter-form-category-li\"><a href='JavaScript:void(0);' onClick=\"mopen('$DivId');\" >$new_category</a><div id='$DivId' ><table border='0' cellspacing='0' cellpadding='0'>";
+          $StringEcho.= "<li class=\"encounter-form-category-li\"><a href='JavaScript:void(0);' onMouseOver=\"mopen('$DivId');\" >$new_category</a><div id='$DivId' onmouseleave=mclose(); ><table border='0' cellspacing='0' cellpadding='0'>";
```
Comment (aethelwulffe) [inline interface/patient_file/encounter/new_form.php]: REal changes in this file only, right?

**Comment by tmccormi** [comment]
This is tagged as a duplicate... of #1202 ? can it be just closed then?

**Pull request closed without merging**